libraries/rib-base/src/main/java/com/badoo/ribs/android/integrationpoint/IntegrationPoint.kt

```diff
@@ -56,6 +57,7 @@ abstract class IntegrationPoint(
         this._root = root
         root.node.integrationPoint = this
         subscribeToLifecycle()
+        if (root is NodeLifecycleAware) root.onAttached()
```

Comment:
Let's do `root.plugins.filterIsInstance<NodeLifecycleAware>().forEach { it.onAttached() }` here. Otherwise behaviour is different for root node and child nodes.

Reply:
Maybe it is even better to create `node.onAttached()` method.

Reply:
Agreed. However, it clashes with the same method name in `NodeLifecycleAware` if `Node` also implements it by delegation via `Connectable`.